### DISCUSSION TITLE
Clone and Edit button

### DISCUSSION THREAD

**Discussion by schlifex** [firefly-iii/firefly-iii#8076] [General]
Most of my transactions are created by searching, cloning and afterwards editing the amount. After having done that thousands of times, I start missing a single button, that clones an existing transaction and opens the page to edit the newly created one.

I know, [this has been requested earlier](https://github.com/firefly-iii/firefly-iii/issues/5167), but I'd like to emphasize the convenience of having that option. In the search list, it adds a fifth choice to the Actions menu next to Edit, Delete, Clone and Create rule, which isn't too confusing in my opinion.

**Comment by JC5** [comment]
It's a great idea for the new layout I'm working on 👍

**Comment by JC5** [comment]
Actually, this was easy and will be part of the next release.

**Reply by matejdro** [reply]
Thanks a lot, this is super helpful to my workflow!

**Reply by glivv** [reply]
Not sure if that's intended, but if you open some transaction (/transactions/show/NNN), do "clone and edit" there, after updating cloned transaction you're returned to the original one, somewhat misleading

**Comment by yeagerghost** [comment]
So, I just checked and saw there was a new version released (6.0.30), checked the new features, saw clone and edit and I was happy.  This is something I have always wanted, but in the meantime, I was editing existing transactions and by muscle memory make sure the FIRST thing I clicked on was "Store as new transaction instead of updating" so that was basically cloning.  It worked out well for me.

Just tried the new clone and edit feature, one difference I see between my old method and the clone and edit method is that an entry is made in the DB immediately.  Sometimes when using "Store new transaction instead of updating" I might change my mind and can always just click away to cancel the "clone" and would have no new entry.  With the clone and edit, once I click the button there is no cancelling :-) 

Just adding my two cents.  I love this product.  Keep up the good work
![store-as-new](https://github.com/firefly-iii/firefly-iii/assets/16521014/b3da15c0-4d6a-4f54-9892-88118041e8e0)
